### ISSUE TITLE
Allow Administrators to Set Default Moderation Rules

### ISSUE THREAD

**Issue by MattyBoombalatty** [JohnXLivingston/peertube-plugin-livechat#308]
In order to mitigate spam and bad actors, Administrators should be able to set automatically blocked/filtered keywords in chatrooms, similar to how individuals can set up their own moderation bots. I don't believe the responsibility should be relegated to the end-user to use regex to stop people from posting certain things in their chats.

**Comment by JohnXLivingston** [comment]
Agree. Currently released moderation tools are just a first shot, and i plan to add some features for instance admins.
I added your issue as a subtask here: https://github.com/JohnXLivingston/peertube-plugin-livechat/issues/116
